misoc/cores/spi_flash.py

```diff
@@ -59,8 +69,12 @@ def __init__(self, pads, dummy=15, div=2, with_bitbang=True):
 
         pads.cs_n.reset = 1
 
-        dq = TSTriple(spi_width)
-        self.specials.dq = dq.get_tristate(pads.dq)
+        if spi_width > 1:
+            dq = TSTriple(spi_width)
+            self.specials.dq = dq.get_tristate(pads.dq)
+        else:
+            dq = SpiIF(pads.miso, pads.mosi, Signal())
+            with_bitbang = False
```

Comment:
Disabling with_bitbang silently like this doesn't sound like a good idea.

Reply:
ok, what about:
```
assert with_bitbang == False, "Bitbang not supported in with 1-bit SPI flash."
```
?